api/src/main/resources/liquibase.xml

```diff
@@ -139,39 +139,39 @@
     <changeSet author="aman (generated)" id="address_hierarchy-1595797202649-7">
         <preConditions onFail="MARK_RAN">
             <not>
-                <foreignKeyConstraintExists foreignKeyName="level_to_level" />
+                <foreignKeyConstraintExists foreignKeyTableName="address_hierarchy_entry" foreignKeyName="level_to_level" />
```

Comment:
To be more explicit about what @mogoodrich said, liquibase does not allow you to modify the contents of an existing changeset once it has been run.  You will need to determine what the original md5sum of these changesets are before your change, and then add those to the top of each changeset before the preConditions.  Something like:
```
        <validCheckSum>3:60fc99e90700bdd6fff363fd5f9a1d02</validCheckSum>
```
You will need to do this for all of these changesets that you change.

For reference if it is helpful, this is what is in one of my sdk databases for addresshierarchy currently:

```
select id, MD5SUM
from liquibasechangelog
where id like '%address_hierarchy-%';
```
Results;
```
20231010-clean-address-hierarchy-address1,8:e2cc90ce8353f8e8e738731884269cef
20231010-clean-address-hierarchy-city_village,8:e6b79243f1ced07c3ee84c13b8dc21d3
20231010-clean-address-hierarchy-county_district,8:0d657dc67169eb381448789940364d91
20231010-clean-address-hierarchy-state_province,8:c44a389f2558846dd4827768d9945dac
address_hierarchy-1595797202649-1,8:7af68c228c63ee24ac3e2c9b91e8bf22
address_hierarchy-1595797202649-2,8:d2ed5a23e363ed1bc853b0dd81aeffeb
address_hierarchy-1595797202649-3,8:212ff915d280852a7155f39826b27d3f
address_hierarchy-1595797202649-4,8:6a51c7db6a958ef729963187f36b57dc
address_hierarchy-1595797202649-5,8:6854e0527dec4660bf44fcfef4db05cd
address_hierarchy-1595797202649-6,8:b966d8fe42054bbc575db80e38374650
address_hierarchy-1595797202649-7,8:9cda37caf56963005b8cf80b29e33bbb
address_hierarchy-1595797202649-8,8:0829ef612906edde3ee09ed23cdf045d
address_hierarchy-1595797202649-9,8:23718917acff5432859dfbf2f408435d
address_hierarchy-1595832956041-1,8:a43e3e0ab172198952209eaba316c225
address_hierarchy-1595832956041-116,8:643a6de0b36c21e7649c72f7f5a13399
address_hierarchy-1595832956041-229,8:2ce0874029567b6d0f1063047c54a820
address_hierarchy-1595832956041-516,8:d0b31e94e752d75c753a7618211053cc
address_hierarchy-1595832956041-625,8:6fc70b53ff0bb150a12b9685fcc47363

```